app/lib/matador/components/job-data/JobDataPanel.tsx

```diff
@@ -0,0 +1,42 @@
+import { Divider, Title, } from "@mantine/core";
+import { NavLink } from "react-router-dom";
+import { JobData } from "./JobData";
+import JobDataPanelProps from "./JobDataPanelProps";
+
+
+export const JobDataPanel = ({ parent, data, opts }: JobDataPanelProps) => {
+    const parentLink =
+        parent !== undefined
+            ?
+            `./../../${JSON.parse(data.job.parent).queueKey.split(":")[1]}/${JSON.parse(data.job.parent).id}`
+            :
+            '#';
+
+    return (
+        <>
+            <Title mb='sm' order={3} sx={(theme) => ({
+                color: theme.colorScheme === 'dark' ? theme.colors.dark[0] : theme.black,
+            })}>
+                Job Data
+            </Title>
+
+            <Divider mb='md' />
+
+            <NavLink to={parentLink}>
+                <JobData
+                    title="Parent"
+                    json={parent !== undefined ? parent : {}}
+                />
+            </NavLink>
+            <JobData
+                title="Opts"
+                json={opts !== undefined ? opts : {}}
+            />
+
+            <JobData
+                title="Data"
+                json={data !== undefined ? data : {}}
+            />
+        </>
+    );
+}
```

Comment:
Others empty lines.

Is prettier who has formatted that ternary operator in this way?

Reply:
I have formatted all with Prettier, so i don't know why there are those blank lines.
No, it was me but in that case after development i have formatted with prettier.
Now i think it's correct